src/Components/SharedComponents/03.Footer/Footer.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+interface Props {
+}
+
+const Footer: React.FC<Props> = () => {
```

Comment:
can you make this component match our other components with props so far for typescript please?